src/apps/cardano/get_public_key.py

```diff
@@ -34,7 +34,9 @@ def _get_public_key(root_node, derivation_path: list):
     public_key = hexlify(seed.remove_ed25519_prefix(node.public_key())).decode()
     chain_code = hexlify(node.chain_code()).decode()
     xpub_key = public_key + chain_code
-    root_hd_passphrase = hexlify(_derive_hd_passphrase(root_node)).decode()
+
+    # In derivation scheme v2 the passphrase is not used
+    root_hd_passphrase = None
```

Comment:
If root_hd_passphrase is not used in v2, should we remove it `CardanoPublicKey`?

Reply:
Well yes it could be removed as for now it is not used. It could be used as another layer of security in future. Should I create new PR for that?

Reply:
I am already working on it. Thanks!